public/js/controllers/ticketpool_controller.js

```diff
@@ -75,14 +76,17 @@ function priceGraphData (items, memP) {
   return p
 }
 
-function getVal (val) { return isNaN(val) ? 0 : val }
-
-function outputsGraphData (items) {
-  return [
-    getVal(items.solo),
-    getVal(items.pooled),
-    getVal(items.txsplit)
-  ]
+function populateOutputs (data) {
+  var totalCount = data.count.reduce((a, n) => { return a + n }, 0)
+  var tableData = '<thead><td># of Outputs</td><td>Count</td><td>% Occurence</td></thead>'
+  data.outputs.map((n, i) => {
+    var count = data.count[i]
+    tableData += '<tr><td class="pr-2 lh1rem vam nowrap xs-w117 font-weight-bold">' + n + '</td>' +
+    '<td><span class="hash lh1rem"> ' + count + ' </span></td>' +
+    '<td><span class="hash lh1rem"> ' + ((count * 100) / totalCount).toFixed(4) + '% </span></td></tr>'
+  })
+  dompurify.sanitize({ IN_PLACE: true }, tableData)
```

Comment:
Your arguments are switched here. And when sanitizing `IN_PLACE`, you must provide a node, not raw HTML. This is why I suggested you might use `parseFloat` and `parseInt` instead of dompurify. Use a template literal, and do something like

```
tableData += `<tr><td class="pr-2 lh1rem vam nowrap xs-w117 font-weight-bold">${parseInt(n)}</td>
    <td><span class="hash lh1rem">${parseInt(count)}</span></td>
    <td><span class="hash lh1rem">${((parseInt(count) * 100) / parseFloat(totalCount)).toFixed(4)}% </span></td></tr>`
```
 
and you won't need `dompurify` at all. You can only use this approach when all of your variables are expected to be numbers. 

If you do want to use dompurify, you'll need to create a node, and then dump the child. i.e.

```
tbody = document.createElement('tbody')
tbody.innerHTML = tableData
dompurify.sanitize(tbody, { IN_PLACE: true })
return tbody.firstChild

...
// And then below
while(this.outputsTarget.firstChild) this.outputsTarget.removeChild(this.outputsTarget.firstChild)
this.outputsTarget.appendChild(populateOutputs(data['outputs_chart']))
```

IMO, using `parseInt` and `parseFloat` to sanitize is easier in this case.